napari/layers/shapes/_tests/test_triangulation.py

```diff
@@ -187,3 +191,116 @@ def test_reconstruct_polygon_edges():
+            [-25.87, 200.76],
+            [-26.48, 200.67],
+            [-26.83, 200.89],
+            [-26.73, 201.61],
+            [-26.28, 202.11],
+            [-25.98, 202.58],
+            [-25.5, 202.82],
+            [-25.27, 203.31],
+            [-25.39, 203.73],
+            [-25.67, 204.21],
+            [-25.72, 205.03],
+            [-25.49, 205.66],
+            [-25.17, 205.77],
+            [-24.7, 205.94],
+            [-24.62, 206.49],
+            [-24.24, 206.79],
+            [-23.57, 207.12],
+            [-22.83, 208.02],
+            [-22.09, 209.43],
+            [-22.1, 209.84],
+            [-22.27, 210.32],
+            [-22.15, 210.66],
+            [-22.25, 211.19],
+            [-23.66, 211.67],
+            [-24.37, 211.93],
+            [-25.48, 211.75],
+            [-25.84, 211.84],
+            [-25.66, 211.33],
+            [-25.73, 211.04],
+            [-26.02, 210.95],
+            [-26.4, 210.68],
+            [-26.74, 210.69],
+            [-27.29, 211.28],
+            [-27.18, 211.87],
+            [-26.73, 212.07],
+            [-26.74, 212.83],
+            [-27.47, 212.58],
+            [-28.3, 212.46],
+            [-28.75, 212.2],
+            [-29.26, 211.52],
+            [-29.4, 211.33],
+            [-29.91, 210.9],
+            [-30.42, 210.62],
+            [-31.14, 210.06],
+            [-32.17, 208.93],
+            [-32.77, 208.22],
+            [-33.23, 207.46],
+            [-33.61, 206.42],
+            [-33.67, 205.91],
+            [-33.94, 205.78],
+            [-33.8, 205.17],
+            [-33.99, 204.68],
+            [-33.79, 203.59],
+            [-33.92, 202.99],
+            [-33.86, 202.57],
+            [-34.26, 201.54],
+            [-34.42, 200.69],
+            [-34.8, 200.07],
+            [-34.82, 199.62],
+            [-34.46, 199.19],
+            [-34.44, 198.86],
+            [-34.0, 198.42],
+            [-34.14, 198.38],
+            [-33.87, 198.24],
+            [-33.28, 198.25],
+            [-32.61, 197.93],
+            [-32.43, 198.25],
+            [-31.66, 198.22],
+            [-30.73, 197.57],
+            [-29.88, 197.06],
+            [-28.58, 196.34],
+            [-28.96, 208.98],
+            [-28.65, 208.54],
+            [-28.85, 208.07],
+            [-29.24, 207.53],
+            [-29.88, 207.0],
+            [-30.65, 207.75],
+            [-30.55, 208.11],
+            [-30.23, 208.29],
+            [-30.07, 208.85],
+            [-29.74, 209.02],
+            [-29.26, 209.33],
+            [-28.96, 208.98],
+        ]
+    )
+
+
+def test_normalize_vertices_and_edges_py_numba_same(country_wth_hole):
+    v1, e1 = ac.normalize_vertices_and_edges(country_wth_hole, close=True)
+    v2, e2 = _accelerated_triangulate_python.normalize_vertices_and_edges_py(
+        country_wth_hole, close=True
+    )
+    e1s = {tuple(x) for x in e1}
+    e2s = {tuple(x) for x in e2}
+    assert len(e1s) == len(e2s)
+    assert e1s == e2s
```

Comment:
```suggestion
    assert e1s == e2s
```

the first assert is redundant?

Reply:
Hm. Technically, yes. Maybe the error message is more readable now. But it is redundant.

Reply:
I just checked and pytest gives a nice set diff anyway:

```pytb
src/affinder/_tests/test_set.py F                                        [100%]
src/affinder/_tests/test_set.py:0 test_set - assert {4, 5} == {5}   …    [100%]
=================================== FAILURES ===================================
___________________________________ test_set ___________________________________

    def test_set():
>       assert {4, 5} == {5}
E       assert {4, 5} == {5}
E
E         Extra items in the left set:
E         4
E         Use -v to get more diff

src/affinder/_tests/test_set.py:2: AssertionError
```